components/concept/awesomebar/src/main/java/mozilla/components/concept/awesomebar/AwesomeBar.kt

```diff
@@ -90,6 +91,7 @@ interface AwesomeBar {
         val title: String? = null,
         val description: String? = null,
         val icon: Bitmap? = null,
+        val indicatorIcon: Drawable? = null,
```

Comment:
nit: Please add a kdoc to the method explaining what this icon is.

Reply:
What should I exactly add kdoc text?
I mean I'm not a native speaker. What's the doc you guys will best understand?

Reply:
I've added kdoc, if you don't like please tell me :)